game.js

```diff
@@ -0,0 +1,235 @@
+class Game {
+    constructor(canvas) {
+        this.canvas = canvas;
+        this.screen = canvas.getContext('2d');
+        this.bodies = [];
+        this.gameSize = {
+            x: canvas.width,
+            y: canvas.height
+        }
+        this.player = new Player(this);
+        this.keyboarder = new Keyboarder();
+        this.snitch = new Snitch(this);
+        this.bludger = new Bludger(this);
+    }
+
+    update() {
+
+        if (colliding(this.snitch, this.player)) {
+            score = score + 1;
+            this.snitch.update();
+            console.log(score)
+        };
+
+        if (colliding(this.player, this.bludger)) {
+            score = 0;
+            // this.bodies.push(bludger);
+            //Destroy both and create new player and continue bludgers...
+            
+        } 
+
+        // else {
+        this.player.update();
+        this.bludger.update();
+        // Code below TESTING
+        this.bodies = this.bodies.concat(new Bludger(this));
+        this.bodies = this.bodies.concat(new Player(this));
```

Comment:
Your collision handling is great, as is calling `.update` on the player and bludger. I'm not sure what you're trying to do by adding a new bludger and player to `this.bodies`, though.